### PR TITLE
Add profile model

### PR DESCRIPTION
**Pull request by MinaKhamesi**
the same profile model for regular users and chefs, if isChef they can modify the recipes field

### CODE REVIEW

File: server/models/recipeModel.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const recipeSchema = new Schema({
+  user: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
```
Comment (jessdesr) [inline server/models/recipeModel.js]: This should probably be associated with a Chef profile, not a user?

**Review by jessdesr — APPROVED** [review]
Nice work, good job iterating with the team.

File: server/models/recipeModel.js (hunk 1 above)
Comment (MinaKhamesi) [reply server/models/recipeModel.js]: Hey @jessdesr , Thanks for your time.
 That makes sense. I saw in the sketch that when we are fetching all recipes, we will want the name of the chef as well, that's why I connected them this way. But now I see we will need location and picture as well, so it makes sense to have it linked with chefProfile. 

Also I think I need to have the name field in the profile too. for rendering the recipes in "browse-meal" we will need that info and getting that separately from user model will be complicated. Actually I cannot see how that will be accomplished.

Comment (jessdesr) [reply server/models/recipeModel.js]: That's a good point.... I think for now we can merge this in, then think about solution for browsing meals

Comment (MinaKhamesi) [reply server/models/recipeModel.js]: Sure. So I will modify the recipe user field to be linked to chefProfile and I will merge.

**Pull request merged**